src/Discord/Parts/Embed/Embed.php

```diff
@@ -469,6 +504,26 @@ public function setURL(string $url): self
         return $this;
     }
 
+    /**
+     * Checks to see if an url is valid to be used in embeds.
+     *
+     * @param ?string $url
+     *
+     * @return bool
+     */
+    protected function checkEmbedUrl(?string $url, array $allowed = ['http', 'https', 'attachment']): bool
```

Comment:
The name `checkEmbedUrl` doesn't really indicate what it's checking. `validateUrlScheme` or something like that would be more descriptive.